### PR TITLE
refactor : converted html to jsx

### PR DESCRIPTION
**Pull request by shubhambishnoi**
Converted all html pages to jsx 
Added Routes, Links
Screenshot from 2022-03-27 22-14-49
Screenshot from 2022-03-27 22-14-53

### CODE REVIEW

File: src/pages/Auth/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+                <input type="text" placeholder="xyz@gmail.com" id="login-email" />
+            </span>
+            <span className="pwd">
+                <label for="login-pwd">Password</label>
+                <input type="password" placeholder="********" id="login-pwd" />
+                <i className="fa-solid fa-eye"></i>
+                <span className="forgot">Forgot your password?</span>
+            </span>
+
+            <div><input type="checkbox" id="remember"/> <label for="remember">Remember me</label></div>
```
Comment (manismk) [inline src/pages/Auth/Login.jsx]: change for to htmlFor

File: src/pages/Auth/Signup.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+                <input type="password" placeholder="********" id="signup-pwd"/>
+                <i className="fa-solid fa-eye"></i>
+            </span>
+            <span className="pwd">
+                <label for="signup-c-pwd">Confirm Password</label>
+                <input type="password" placeholder="********" id="signup-c-pwd"/>
+                <i className="fa-solid fa-eye"></i>
+            </span>
+
+            <div><input type="checkbox" id="signup-accept"/> <label required for="signup-accept">I accept all Terms &
```
Comment (manismk) [inline src/pages/Auth/Signup.jsx]: change for to htmlFor

File: src/pages/ProductListing/ProductListing.jsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+      <div>
+        <input name="sort" type="radio" id="l-h"/>{" "}
+        <label for="l-h">Price - Low to High</label>
+      </div>
+
+      <div>
+        <input name="sort" type="radio" id="h-l"/>{" "}
+        <label for="h-l">Price - High to Low</label>
+      </div>
```
Comment (manismk) [inline src/pages/ProductListing/ProductListing.jsx]: change for to htmlFor

**Review by manismk — APPROVED** [review]
Good folder structure.

Remove the empty landing CSS if not used

File: src/pages/Auth/Signup.jsx (hunk 2 above)
Comment (Tanay-J) [inline src/pages/Auth/Signup.jsx]: use ```htmlFor``` here since ```for``` is a reserved keyword in js

**Review by Tanay-J — APPROVED** [review]
The original HTML code has already been reviewed in earlier projects. 

Code re-usability can be looked after (eg. using utility classes for ```padding```, ```margin```)

Good work! Approving the changes.

File: src/pages/Auth/Signup.jsx (hunk 2 above)
Comment (shubhambishnoi) [reply src/pages/Auth/Signup.jsx]: done

File: src/pages/ProductListing/ProductListing.jsx (hunk 3 above)
Comment (shubhambishnoi) [reply src/pages/ProductListing/ProductListing.jsx]: done

File: src/pages/Auth/Signup.jsx (hunk 2 above)
Comment (shubhambishnoi) [reply src/pages/Auth/Signup.jsx]: made the changes. Thanks !

**Pull request merged**